Dockerfile

```diff
@@ -1,100 +1,53 @@
-# Ripped off from Avalon
-# Base stage for building gems
-FROM        ruby:2.7.2-buster as bundle
-RUN         echo "deb http://deb.debian.org/debian buster-backports main" >> /etc/apt/sources.list \
-         && apt-get update \
-         && rm -rf /var/lib/apt/lists/* \
-         && apt-get clean
-
-COPY        lib/hyku_addons/version.rb ./lib/hyku_addons/version.rb
-COPY        hyku_addons.gemspec ./hyku_addons.gemspec
-COPY        Gemfile ./Gemfile
-COPY        Gemfile.lock ./Gemfile.lock
-COPY        spec/internal_test_hyku/Gemfile ./spec/internal_test_hyku/Gemfile
-COPY        spec/internal_test_hyku/Gemfile.lock ./spec/internal_test_hyku/Gemfile.lock
-
-RUN         gem install bundler -v "$(grep -A 1 "BUNDLED WITH" Gemfile.lock | tail -n 1)" \
-         && bundle config build.nokogiri --use-system-libraries
-
-
-# Build development gems
-FROM        bundle as bundle-dev
-RUN         bundle config set without 'production'
-RUN         bundle config set with 'aws development test postgres'
-ENV         CFLAGS=-Wno-error=format-overflow
-RUN         bundle install --jobs=4 --retry=3
-
-
 # Base stage for building final images
-FROM        ruby:2.7.1-slim-buster as base
-
-RUN         apt-get update && apt-get install -y --no-install-recommends curl gnupg2 ffmpeg \
-         && curl -sL http://deb.nodesource.com/setup_8.x | bash -
-RUN         apt-get update && apt-get install -y --no-install-recommends --allow-unauthenticated \
-            nodejs \
-            sendmail \
-            git \
-            libxml2-dev \
-            libxslt-dev \
-            openssh-client \
-            zip \
-            dumb-init \
-            default-jre \
-            # Below copied from hyku's Dockerfile
-            build-essential \
-            ghostscript \
-            imagemagick \
-            libpq-dev \
-            libreoffice \
-            libsasl2-dev \
-            netcat \
-            postgresql-client \
-            rsync \
-            tzdata \
-            unzip \
-            && \
-            apt-get clean && \
-            rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
-
-# If changes are made to fits version or location,
-# amend `LD_LIBRARY_PATH` in docker-compose.yml accordingly.
+FROM phusion/passenger-ruby27:2.0.0 as BASE_IMAGE
+
+RUN install_clean --allow-unauthenticated \
```

Comment:
So the missing packages from the old dockerfile have been pre-installed ?

      openssh-client ,  tzdata, nodejs, curl gnupg2 ffmpeg

Reply:
`gnupg2 ffmpeg` are there, just lower down the list. The others are preinstalled. I also added Vim because its annoying having to install it to debug something inside a Gem. 

The installed packages can be listed with `dpkg -l` or ` apt list --installed`, to confirm the packages which might be mising from the list. :)